### PR TITLE
Fix-up NPM cache for UI CICD job

### PR DESCRIPTION
**Pull request by Willdotwhite**
Irritatingly, couldn't test this one before merging - whoops

### CODE REVIEW

File: .github/workflows/ui-run-tests.yml [hunk 1]
```diff
@@ -36,7 +36,6 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-          cache: 'npm'
```
Comment (dallonf) [inline .github/workflows/ui-run-tests.yml:39]: What issue are you running into? Using the cache will dramatically speed up PR checks so I'm hesitant to disable.

Comment (Willdotwhite) [reply .github/workflows/ui-run-tests.yml:39]: Don't have the lockfile set up in the job yet; I want to quickly drop the cache so the job runs, get it passing, and then add the cache after the fact if possible. Happy to approach it from the other side.

Failing job example: https://github.com/GameMakersToolkit/team-finder/runs/3839966779?check_suite_focus=true

Comment (Willdotwhite) [reply .github/workflows/ui-run-tests.yml:39]: Wait, I'm an idiot, turns out `cache-dependency-path` was what I was missing

Comment (Willdotwhite) [reply .github/workflows/ui-run-tests.yml:39]: @dallonf have added the cache in correctly, but the runs that were no-cache (run #2) and cache (run #4) took about the same time. Looks like there's a bit more work to do, bear with

Comment (Willdotwhite) [reply .github/workflows/ui-run-tests.yml:39]: Appears after a successful run
image

**Review by dallonf — APPROVED** [review]

**Pull request merged**